app/views/sessions/show.html.erb

```diff
@@ -47,6 +49,24 @@
         </p>
       </div>
     </div>
+
+    <% unless Rails.env.production? %>
+      <%= govuk_details(
+        summary_text: "Dev tools (only available in non-production environments)"
+      ) do %>
+        <% if Flipper.enabled? :make_session_in_progress_button %>
+          <p>
+            Nurses can only record vaccinations while a session is "in-progress". You can
+            make this session "in-progress" to test vaccination recording.
+          </p>
+          <%= govuk_button_to(
+            "Make session in-progress",
+            make_in_progress_session_path,
+            method: :put
+          ) %>
```

Comment:
Maybe something like this, makes it a bit clearer that the in progress state only lasts for a day:

```suggestion
          <p>
            Vaccinations can only be recorded if a session is in progress. If you want
            to test vaccination recording, you can set this session as in progress for today.
          </p>
          <%= govuk_button_to(
            "Set session in progress for today",
            make_in_progress_session_path,
            method: :put
          ) %>
```